packages/sampler/stories/qa/Tooltip.js

```diff
@@ -59,7 +59,7 @@ class ChangeableTooltip extends React.Component {
 	}
 
 	changeTooltipText = () => {
-		const {stringText} = this.state;
+		const {text: stringText} = this.state;
```

Comment:
Odd that we'd rename the property `stringText` rather than just change the references to the actual name of the state (`text`)

Reply:
`text` is defined already in the module scope for the knob so this avoids shadowing